content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+
+- Download your ```kubeconfig``` file, and define the Kubeconfig file as a shell environment variable
+
+```bash
+# Example
+export KUBECONFIG="/the/path/of/your/kubeconfig"
+```
+
+#### Using REST API
+HPE Ezmeral Container Platform provides a REST API for you to interact with. Here is the set of REST API calls that allows you to download the Kubeconfig file
+
+```bash
+# Authenticate as a tenant user in the specified tenant, getting the session ID:
+curl -k -i -s --request POST "http://ez53-gateway.hpeilab.com:8080/api/v2/session" \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+"name": "username",
+"password": "password",
+"tenant_name": "test-tenant"
+}'
+
+# output
+HTTP/1.1 201 Created
+Access-Control-Allow-Origin: *
+Content-Length: 13
+Content-Type: text/plain
+Date: Fri, 30 Apr 2021 13:18:38 GMT
+Location: /api/v2/session/__thisisthesessionid__
+Server: HPE Ezmeral Container Platform 5.3
+
+201 Created
+```
+
+Get the Kubeconfig file for your tenant working context: 
+
+```bash
+curl -k -s --request GET "http://ez53-gateway.hpeilab.com:8080/api/v2/k8skubeconfig" \
+--header "X-BDS-SESSION: /api/v2/session/__thisisthesessionid__" \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' > ./kubeconfig
+
+# Define the Kubeconfig file as a shell environment variable
+export KUBECONFIG=kubeconfig
+```
+The screenshot below shows you how you can combine two commands into a single command:
+
+```bash
+curl -k -s --request GET "http://<you-ez-gateway>:8080/api/v2/k8skubeconfig" \
+--header "X-BDS-SESSION: $(curl -k -i -s --request POST "http://<you-ez-gateway>:8080/api/v2/session" \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+"name": "<change-your-user-name>",
+"password": "<change-your-user-password>",
+"tenant_name": "<change-the-tenant-you-want>"
+}' | grep Location | awk '{print $2}' | tr -d '\r')" \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' > ./kubeconfig
+
+export KUBECONFIG="./kubeconfig"
+
+kubectl get pods
+```
+
+> Resources:
+> - [API_Access](https://docs.containerplatform.hpe.com/53/reference/accessing-the-applications/API_Access.html)
+> - [Jupyter Notebook: Introduction to the HPE Ezmeral Container Platform REST API](https://github.com/HewlettPackard/hpe-notebooks/tree/master/HPECPAPI)
+> - [API documents](https://github.com/bluedatainc/solutions/tree/master/APIs)
+> - [HPE Container Platform REST API](https://developer.hpe.com/blog/hpe-container-platform-rest-api-part-1-authenticating/)
+
+## 4. hpecp python library (pre-alpha)
+If you are looking for a way to interact with HPE Ezmeral programmatically, you can keep an eye on the hpecp python library from HPE Container Platform Community. Note that it is still a prototype, it may be unstable and subject to change until this library reaches beta.
+
+> Reference:

```

Comment:
References: (with an 's' at the end because there are mote than one items in the reference list. 
The list items for the references appear in a single line in the rendered blog post. Please make sure you insert a line break after "References:"

Reply:
What do you mean by a line break??

---

like this?